root/static/scripts/common/constants/locales.json

```diff
@@ -421,15 +489,23 @@
   "ig_NG":"Igbo Nigeria",
   "ii":"Sichuan Yi",
   "ii_CN":"Sichuan Yi China",
+  "io":"Ido",
+  "io_001":"Ido world",
   "is":"Icelandic",
   "is_IS":"Icelandic Iceland",
   "it":"Italian",
   "it_CH":"Italian Switzerland",
   "it_IT":"Italian Italy",
   "it_SM":"Italian San Marino",
   "it_VA":"Italian Vatican City",
+  "iu":"Inuktitut",
+  "iu_CA":"Inuktitut Canada",
+  "iu_Latn":"Inuktitut Latin",
+  "iu_Latn_CA":"Inuktitut Canada Latin",
   "ja":"Japanese",
   "ja_JP":"Japanese Japan",
+  "jbo":"Lojban",
+  "jbo_001":"Lojban world",
```

Comment:
I'm confused about this one, but I guess it is what it is?

Reply:
Yeah, but if it's the only one... How is it different from the top one? :)

Reply:
I don’t know. As mentioned below, it’s probably worth investigating on its own as it isn’t new either: `eo_001` and `eo`, `et_EE` and `et`…